jsx/DataTable.js

```diff
@@ -440,7 +440,7 @@ class DataTable extends Component {
                 // Note: Can't currently pass a key, need to update columnFormatter
                 // to not return a <td> node. Using createFragment instead.
                 // let key = 'td_col_' + j;
-                curRow.push(cell);
+                curRow.push(React.cloneElement(cell, {key: 'td_col_' + j}));
```

Comment:
This line solves `[Error] Warning: Each child in a list should have a unique "key" prop.` and the React.cloneElement just clones the cell with passing a key to it. The prior existing key was null.